### ISSUE TITLE
Better handling of rides assigned to drivers by dispatch

### ISSUE THREAD

**Issue by john** [john/drive.vote#1123]
In CA-25 we saw cases were dispatch would assign a ride to a driver, who would then go and get the voter without first accepting the ride, which left the dispatch view in a confusing state where the voter was showing as 'Needs driver' even though they had one assigned who was going to get them.

**Comment by dryan** [comment]
yeah that's right, I'm thinking of a "Pending driver acceptance" state for this case
